blocks/library/button/index.js

```diff
@@ -182,13 +174,13 @@ export const settings = {
 
 	attributes: blockAttributes,
 
-	getEditWrapperProps( attributes ) {
-		const { align, clear } = attributes;
-		const props = {};
+	supports: {
+		align: true,
+		wideAlign: false,
```

Comment:
Why `wideAlign: false`? Seems to be allowed in master.

Reply:
It's broken in master though :) Styles aren't applied for `full` and `wide` properly... We can fix that.

Reply:
I don't think it is designed to work with the theme option `alignWide`.

By the way, we should rename `wideAlign` to `alignWide` to match with PHP.

Reply:
This is how `wide` looks like in action:
![screen shot 2018-02-19 at 11 51 19](https://user-images.githubusercontent.com/699132/36380559-07f5a40e-1583-11e8-958c-6b1508cfea02.png)
![screen shot 2018-02-19 at 11 51 30](https://user-images.githubusercontent.com/699132/36380560-08166db0-1583-11e8-81d3-e721e47ebbe3.png)

Let's leave it disabled for the time being. @karmatosed @jasmussen any plans to make button block wide aligned?